ui/scripts/network.js

```diff
@@ -1100,11 +1100,23 @@
                                         });
                                         args.$form.find('.form-item[rel=cleanup]').find('input').attr('checked', 'checked'); //checked
                                         args.$form.find('.form-item[rel=cleanup]').css('display', 'inline-block'); //shown
+                                        args.$form.find('.form-item[rel=makeredundant]').find('input').attr('checked', 'checked'); //checked
+                                        args.$form.find('.form-item[rel=makeredundant]').css('display', 'inline-block'); //shown
+
+                                        if (Boolean(args.context.networks[0].redundantrouter)) {
```

Comment:
As I understand, for RVR also the rolling upgrade will be applicable, any reason we hide the button.

Reply:
This code will show option to make network redundant during restart, we hide the checkbox/label for networks that are already redundant.

Reply:
@rhtyd, When we say redundant networks, that means it already contains the new upgraded VR(after Blue-Green deployment), we created after a restart? If that is the case, I think it would be great, if we give a different name instead of redundant. Because there is a chance we will misinterpret as RVR.

Reply:
@nitin-maharana I'm not sure where the confusion is, a redundant network already has rVRs (backup+master) therefore no need to show the label/checkbox to user. This is same as the option you get when you restart VPC (see try that, restart VPC with 4.11+).

Reply:
@nitin-maharana I'm not sure what the confusion is. The UI shows the option to makeRedundant a network which is not redundant, otherwise the checkbox is not shown. This is same as what you see in case of VPC restart button/dialog. Thanks.